sidebars.js

```diff
@@ -93,12 +92,30 @@ const sidebars = {
                   type: 'doc',
                   id: 'integration-guides/next-auth',
                   label: 'NextAuth',
-                }, 
+                },
              ],
             },
 */
           ],
         },
+        {
+          type: 'category',
+          label: 'Session Management',
+          collapsed: true,
+          collapsible: true,
+          link: {
+            type: 'doc',
+            id: 'authentication/session-management',
+          },
+          items: [
+            'authentication/session-management-overview',
+            'api-tokens/create-api-token',
+            'api-tokens/validate-access-tokens',
+            'api-tokens/list-access-tokens',
+            'api-tokens/revoke-access-tokens',
+            'api-tokens/enable-refresh-tokens',
```

Comment:
I think linking to the pages is sufficient. I didn't realize they were duplicates at first so I think it might be confusing for other developers. While reading the content you can navigate to them with the link.

Reply:
Done 👍